applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -10,67 +18,102 @@ def Factory(settings, model):
 
 
 class ComputeDragProcess(KratosMultiphysics.Process):
```

Comment:
I think it would make sense to use „Base“ in the name to make clear that it cannot be used standalone

Reply:
I followed the example of fluid. But fluid_solver also does not have base (anymore) in the naming.

Reply:
Updated instead the description of the base class.

Reply:
Ok